sdk/batch/Azure.ResourceManager.Batch/src/Generated/Models/BatchIPRule.cs

```diff
@@ -34,7 +34,7 @@ internal BatchIPRule(BatchIPRuleAction action, string value)
         }
 
         /// <summary> Action when client IP address is matched. </summary>
-        public BatchIPRuleAction Action { get; set; }
```

Comment:
why doesn't this show up as an apiview change?  This is GA so we will need to write custom code for this

There are a quite a few of these can we take a stab at fixing all the back compat issues for GA'd code?  Will we do this with configuration or hand written custom code?